.drone.yml

```diff
@@ -30,7 +30,7 @@ steps:
       filename: unit-tests-cache.tar
 
   - name: unit-tests
-    image: codedotorg/code-dot-org:1.5.2
+    image: codedotorg/code-dot-org:bundler-2.3.22
```

Comment:
We need to generate a new signature and place it at the end of the updated .drone.yml file to ensure our Drone Server trusts it and executes it.

Reply:
Yep; we'll also want to build and tag a new mainline 1.5.3 version